books/free-programming-books-langs.md

```diff
@@ -2143,6 +2143,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Building REST APIs with Rails](https://www.softcover.io/read/06acc071/api_on_rails) - Abraham Kuri Vargas
 * [Essential Ruby on Rails](https://www.programming-books.io/essential/rubyonrails/) - Krzysztof Kowalczyk, StackOverflow Contributors
 * [Kestrels, Quirky Birds, and Hopeless Egocentricity](https://leanpub.com/combinators/read) - Reg Braithwaite
+* [Learn Ruby on Rails: Book One](https://www.softcover.io/read/e8898d1d/learn-rails-1) - Daniel Kehoe
```

Comment:
why not https://learn-rails.com/content/learnrails1/

Reply:
Didn't know about this! Thanks

Changed 👍🏼